pypose/lietensor/basics.py

```diff
@@ -122,6 +122,86 @@ def add(input, other, alpha=1):
     '''
     return input.add(other, alpha)
 
+def mul(input, other):
+    r'''
+    Multiplies input LieTensor by other.
+
+    .. math::
+        \bm{y}_i =
+        \bm{x}_i \times \bm{a}_i
+
+    where :math:`\bm{x}` is the ``input`` LieTensor, :math:`\bm{a}` is the ``other`` value,
+    and :math:`\bm{y}` is the output value.
+    
+    :obj:`LieTensor` @ :obj:`LieTensor` -> :obj:`LieTensor` (group multiplication) 
+        
+    :obj:`LieTensor` @ :obj:`Tensor` -> :obj:`Tensor` (group action)
```

Comment:
You need to clearly define different situations, refer to the doc of [Exp](https://pypose.org/docs/main/generated/pypose.Exp/)